### PR TITLE
Add --unique option to support spritezero's generateLayoutUnique

### PR DESCRIPTION
**Pull request by vdumont**
Add --unique option to support spritezero's `generateLayoutUnique`.
Also regenerate fixtures since they were stale. ie: A fresh copy of origin/master fails to run tests since it picks up spritezero 3.5+ which switched to shelf-pack instead of bin-pack.

### CODE REVIEW

**Comment by vdumont** [comment]
@bhousel This is the companion patch that adds `--unique` option to the CLI to leverage `generateLayoutUnique` (https://github.com/mapbox/spritezero/commit/00dd132bc36b2a283afbf0a453050ac3ff0f5759)

File: bin/spritezero [hunk 1]
```diff
@@ -41,11 +42,14 @@ if (argv.help || argv._.length < 2) {
 }
 
 var ratio = 1;
+var unique = false;
 
 if (argv.retina) {
   ratio = 2;
 } else if (argv.ratio) {
   ratio = parseFloat(argv.ratio);
+} else if (argv.unique) {
```
Comment (tmcw) [inline bin/spritezero]: This shouldn't be an exclusive option to argv.ratio - else if will mean that if `argv.ratio` is defined, the argv.unique option will not have an effect, which is incorrect.

Comment (vdumont) [inline bin/spritezero]: Good catch!

**Comment by vdumont** [comment]
@bhousel @tmcw Can you guys take a look when you have a minute?

**Comment by bhousel** [comment]
Looks good @vdumont, thanks!

**Comment by vdumont** [comment]
@bhousel Awesome, thanks much!

**Comment by vdumont** [comment]
@bhousel Sorry forgot one thing, any chance you can tag this for a new release?

**Comment by bhousel** [comment]
@vdumont yes, already done - it's v1.1.0

Need @tmcw to grant permission for me to npm publish it...

**Comment by vdumont** [comment]
Awesome, thank you so much! 👍

**Pull request merged**